patient-management-service/README.md

```diff
@@ -0,0 +1,100 @@
+# Patient Management Service (Mediflow) - Ballerina
+
+Sample for Patient Management Service service.
```

Comment:
Would be good to give a meaningful intro here. Similar to https://github.com/wso2/choreo-samples/tree/e641ad1480f18bb11f5fbb7039cb1ba12299e105/employee-details-service#introduction